### DISCUSSION TITLE
Agenda

### DISCUSSION THREAD

**Discussion by Godo2000** [Lioydiano/Classeviva#20] [Domande e Risposte]
Ciao, stavo cercando un metodo per automatizzare la creazione di eventi nel calendario che corrispondano a futuri compiti o verifiche nell’agenda di ClasseViva. La mia idea era quella di creare un Comando Rapido (su un mio dispositivo Apple) che venisse eseguito ogni giorno e che, ad ogni avvio, utilizzasse un un applicazione di terze parti (A-Shell, per la precisione) per avviare il programma python per ottenere le informazioni necessarie. Ho letto tutta la documentazione necessaria di ClasseViva.py prima di provare a creare il file python ma non ho ancora avuto fortuna: non tanto per una difficoltà di A-Shell nel comprendere le istruzioni impartire (non gira una versione custom di python e le repository sono quelle ufficiali di pip) quanto più per una mia mancata comprensione della documentazione fornita. Chiedo, se possibile, una mano per creare un semplicissimo file python che mi permetta di ottenere gli eventi futuri di Classe Viva utilizzando la sopracitata libreria e di stamparli con la funzione print(). Grazie in anticipo

**Comment by FLAK-ZOSO** [comment]
Non mi era arrivata nessuna notifica, comunque non ho capito bene la richiesta: per ottenere gli eventi futuri dell'agenda, non sarebbe sufficiente utilizzare questo metodo?

https://github.com/Lioydiano/Classeviva/blob/7a8d727cd29471aea731b59a68a999c79ce351c5/src/classeviva/main.py#L197
